### PR TITLE
post: Bourbon family on GitHub static pages

### LINKED ISSUE #365: Borges's next post 23/03
Houston we have a problem!
The only way to save the world is if Borges post until 23/03.

Have fun ;)

### PR DESCRIPTION
**Pull request by thiagogabriel**
Ready to review and merge

Closes #365

### CODE REVIEW

**Comment by thiagogabriel** [comment]
@beatrizcp87 :dizzy:

File: _posts/2015-03-26-bourbon-family-on-github-pages.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+@import "base/base",
+        "bourbon/bourbon",
+        "neat/neat"
+;
+{% endhighlight %}
+
+## Why does it matter?
+
+Bourbon is a mixin library for Sass. It is much better than Twitter Bootstrap, for example, because you don't need to include tons of useless code. Bourbon family encourages you to write modular and well written Sass. This is the best way I know to write Sass in a productive way and keep the code clean. Even the grid system is great, where you don't need to add `.col-xs-4` or `.grid_2` on your HTML, it is just a matter of add mixins like this in your Sass:
+
```
Comment (alexbrahastoll) [inline _posts/2015-03-26-bourbon-family-on-github-pages.markdown]: A suggestion to make part of this paragraph a bit easier to read:

Even the grid system is great, not requiring you to add classes like `.col-xs-4` or `.grid_2` on your HTML, making its usage just a matter of add mixins like the following in your Sass:

File: _posts/2015-03-26-bourbon-family-on-github-pages.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+author: Thiago Borges
+categories:
+  - thiago borges
+  - github
+  - sass
+  - jekyll
+---
+
+Some weeks ago I got impressed with [proteus](http://thoughtbot.github.io/proteus/) marketing. HAML, Coffeescript, Sass, Bourbon, Neat and Bitters on the project setup.
+
```
Comment (alexbrahastoll) [inline _posts/2015-03-26-bourbon-family-on-github-pages.markdown]: Suggestion to make the last part of this paragraph clearer to the reader:

It allows you to create a new project using HAML, Coffeescript, Sass, Bourbon, Neat and Bitters right from the beginning.

File: _posts/2015-03-26-bourbon-family-on-github-pages.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+<!--more-->
+
+It looked really nice, but I didn't like the workflow deploying a different git branch for generated code and found a better solution to deploy static pages on GitHub pages.
```
Comment (alexbrahastoll) [inline _posts/2015-03-26-bourbon-family-on-github-pages.markdown]: It looked really nice, but I didn't like the workflow: creating a different git branch for the generated code results in some inconveniences. After researching a little bit, I have found what I consider a better solution to deploy static websites on GitHub pages.

File: _posts/2015-03-26-bourbon-family-on-github-pages.markdown [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+GitHub pages is a great and free way to host static pages using Jekyll, a static site generator made with Ruby, or pure HTML. When GitHub pages started [supporting Jekyll 2.2.0](https://github.com/blog/1867-github-pages-now-runs-jekyll-2-2-0), it also started compiling the .coffee, .sass and .scss during the build after `git push`, simplifying the development flow for complex websites which required solutions like [guard-sass](https://github.com/hawx/guard-sass), [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/).
+
+Since GitHub supports Sass, it became quite simple to add Bourbon family ([Bourbon](http://bourbon.io/), [Neat](http://neat.bourbon.io/) and [Bitters](http://bitters.bourbon.io/)) to a static page on GitHub Pages.
```
Comment (alexbrahastoll) [inline _posts/2015-03-26-bourbon-family-on-github-pages.markdown]: Since GitHub started supporting Saas, ...

...the Bourbon family...

**Comment by thiagogabriel** [comment]
@beatrizcp87 :shipit: :rocket:

**Pull request merged**